.circleci/config.yml

```diff
@@ -239,6 +239,11 @@ jobs:
           name: Check if the test should run
           command: |
             ./scripts/ci_check_if_test_should_run_v2.sh @celo/protocol
+      - run:
+          name: Opcode tests
+          command: |
+            sudo apt-get install ripgrep
```

Comment:
I'm gonna say this is optional cause Docker sucks, but consider adding this to the docker image instead of installing in here